crates/symbolicator-service/tests/integration/snapshots/integration__sourcemap__malformed_abs_path.snap

```diff
@@ -17,5 +17,5 @@ raw_stacktraces:
         colno: 1
 errors:
   - abs_path: http//example.com/test.min.js
-    type: missing_source
+    type: scraping_disabled
```

Comment:
this is a bit weird, but actually correct because we check for scraping before we validate the `Url`, right?

Reply:
Yes, mostly because it's more convenient this way. I'm sure we could also do it the other way around if you think that'd be better.